### ISSUE TITLE
unable find home directory or any other linux directories 

### ISSUE THREAD

**Issue by gyani73** [pythops/jetson-nano-image#57]
after building the image and successfully flashing it into the sd card. after booting I entered the default user name and password which gave terminal access (pythops@localhost). I was unable to find directories like home, bin , etc .. by typing the "ls" command. i don't know why?

**Comment by pythops** [comment]
Can you rephrase it ? Not sure If I understand the issue 
How did you log in ? monitor or with ssh ?
provide some screenshots as well.

**Comment by gyani73** [comment]
[](url)
